cornflow-server/cornflow/commands/permissions.py

```diff
@@ -4,24 +4,43 @@
 from cornflow.shared.const import (
     BASE_PERMISSION_ASSIGNATION,
     EXTRA_PERMISSION_ASSIGNATION,
+    ALL_DEFAULT_ACTIONS,
+    ALL_DEFAULT_ROLES,
 )
-from cornflow.models import ViewModel, PermissionViewRoleModel
+from cornflow.models import ViewModel, PermissionViewRoleModel, RoleModel
 from cornflow.shared import db
+
 from flask import current_app
 from sqlalchemy.exc import DBAPIError, IntegrityError
+from cornflow.endpoints import resources, alarms_resources
 
 
 def register_base_permissions_command(external_app: str = None, verbose: bool = False):
     if external_app is None:
-        from cornflow.endpoints import resources, alarms_resources
         resources_to_register = resources
+        extra_permissions = EXTRA_PERMISSION_ASSIGNATION
         if current_app.config["ALARMS_ENDPOINTS"]:
             resources_to_register = resources + alarms_resources
     elif external_app is not None:
         sys.path.append("./")
         external_module = import_module(external_app)
-        resources_to_register = external_module.endpoints.resources
+        try:
+            extra_permissions = (
+                EXTRA_PERMISSION_ASSIGNATION
+                + external_module.shared.const.EXTRA_PERMISSION_ASSIGNATION
+            )
+        except AttributeError:
+            extra_permissions = EXTRA_PERMISSION_ASSIGNATION
+
+        if current_app.config["ALARMS_ENDPOINTS"]:
+            resources_to_register = (
+                external_module.endpoints.resources + resources + alarms_resources
+            )
+        else:
+            resources_to_register = external_module.endpoints.resources + resources
+
     else:
+        # Not reachable code
```

Comment:
If it's not reachable, it should probably be deleted, and the "elif" replaced with an "else"